pkg/projectfile/projectfile.go

```diff
@@ -1005,7 +1005,7 @@ func createCustom(params *CreateParams, lang language.Language) (*Project, error
 	}
 
 	content := params.Content
-	if content == "" && lang.String() != "" {
+	if content == "" && lang != language.Unset {
```

Comment:
The old version seems less error-prone to me. My main concern with raising this code as is that we understand why it broke in the first place. I have no issues with the original code in and of itself.

Case in point; what if in the future we introduce a regression of some sort where it might send a `language.Unknown`.

Reply:
Can we create a unit test that would catch a regression? (I don't know if this is possible/feasible/reasonable. It's simply a question.)

Reply:
I'm not sure the unit test would capture a regression here as the language would be coming from outside of the package.

I've reverted the change to the original. There is a story filed to deal with this bug so if there is a better way we can handle it there. This should at least ensure the affected workflow isn't "broken"